src/dnx/TraceWriter.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "xplat.h"
+
+class TraceWriter
+{
+public:
+    TraceWriter(bool verbose) : m_verbose(verbose)
+    {}
+
+    void Write(const dnx::char_t* entry, bool verbose)
```

Comment:
cppreference is down - is cout::operator << const? I did not want to make it const because semantically this function is changing the environment.